src/common/editable/index.js

```diff
@@ -457,6 +457,7 @@ export class Size extends Editable {
     const { value } = this.props
 
     return <span
+      className='form-inline'
```

Comment:
`SizeInput` uses `input-group` from Bootstrap, which adds `display: table` to the element. So it will behave as a block element. That's why I need `form-inline` on the parent if I want to use it in a span.